src/main/resources/db/migration/V1_13__Create_case_data_tables.sql

```diff
@@ -0,0 +1,59 @@
+create table if not exists darts.moj_reporting_restrictions
+(
+  moj_rer_id      integer,
+  rer_description character varying,
+  constraint reporting_restrictions_type_pkey primary key (moj_rer_id)
+);
+create sequence if not exists moj_rer_seq;
+
+
+create table if not exists darts.moj_case
+(
+  moj_cas_id         integer not null primary key,
+  moj_rer_id         integer
+    constraint moj_case_reporting_restriction_fk references darts.moj_reporting_restrictions (moj_rer_id),
+  r_case_object_id   varchar(16),
+  c_case_id          varchar,
```

Comment:
Should we be ensuring we try to use the same type throughout, we seem to prefer `CHARACTER VARYING` instead of `VARCHAR`? (Not sure why)